src/lib/components/MobileNav.svelte

```diff
@@ -7,22 +7,40 @@
     import { Button, InlineTag, Icon } from '$lib/components/ui';
     import { GithubStats } from '$lib/components/shared';
     import { trackEvent } from '$lib/actions/analytics';
+    import { browser } from '$app/environment';
+    import { onMount } from 'svelte';
 
     export let open = false;
     export let links: NavLink[];
     export let offerButton = false;
 
+    let bannerHeight = 0;
+
     afterNavigate(() => {
         open = false;
         document.body.style.overflow = '';
     });
+
+    function updateBannerHeight() {
+        if (browser) {
+            const banner = document.getElementById('hackathon-banner');
+            bannerHeight = banner ? banner.getBoundingClientRect().height : 0;
+        }
+    }
+
+    onMount(() => {
+        updateBannerHeight();
+        window.addEventListener('resize', updateBannerHeight);
```

Comment:
we already have `svelte:window`, lets use that to avoid managing the adding/removing listener 👍. Svelte automatically takes care of removing listeners.